packages/hooks/src/index.ts

```diff
@@ -1,5 +1,5 @@
-export { useWallet, useContract } from './hooks';
-export { useTokenBalance } from './hooks/useTokenBalance';
-export { Provider, Web3Context } from './Provider';
-export type { ProviderProps, Web3ContextType } from './Provider';
-export { NETWORKS, CHAIN_ID_TO_NETWORK } from './constants';
+export {useWallet, useContract} from './hooks';
+export {useTokenBalance} from './hooks/useTokenBalance';
+export {Provider, Web3Context} from './Provider';
+export type {ProviderProps, Web3ContextType} from './Provider';
+export {NETWORKS, CHAIN_ID_TO_NETWORK, ERC20ABI} from './constants';
```

Comment:
All of these lines should not be changed. Only the last line is actually being changed. Can you reset these?

Reply:
This must be also the result of the post commit hook. I reverted the first few lines just to notice that they have changed again after I committed my changes. 
I can use -no--verify again to restore the first 5 lines. But maybe it's intended that the commit hook applies certain formatting rules to each file even the affected file has nothing to do with the pr

Reply:
I see. No worries then.